src/disco/components/Addon/styles.scss

```diff
@@ -126,8 +126,8 @@ $addon-padding: 20px;
 
     .copy {
       flex-grow: 1;
+      overflow-wrap: break-word;
```

Comment:
Should this one be removed too? It seems like we've removed most (if not all) of these, so why does this one remain?

Reply:
mm can't remember why we have that. I will double check before merging tomorrow.

Reply:
ah wait no, it's the disco app. I did not want to take any risk.

Reply:
the global style is for the amo app only and that's why I kept this one. The disco app will be deleted soon enough so I think it's fine.